pages/common/zig.md

```diff
@@ -0,0 +1,36 @@
+# zig
+
+> The Zig compiler and toolchain.
+> More information: <https://ziglang.org/>.
+
+- Compile the project in the current directory:
+
+`zig build`
+
+- Compile and run the project in the current directory:
+
+`zig build run`
+
+- Initialize a `zig build` application:
+
+`zig init-exe`
+
+- Intitialize a `zig build` libray:
```

Comment:
```suggestion
- Initialize a `zig build` library:
```

Reply:
Hah, great catch. Getting this change in. 🙏

Reply:
Updated in: b791dc6.